### PR TITLE
fix: change teacher dashboard tab design to standard tabs

### PR DESCRIPTION
**Pull request by jfdoming**
## Notion ticket link

[Update Teacher Dashboard test sessions to use existing tab design](https://www.notion.so/uwblueprintexecs/Update-Teacher-Dashboard-test-sessions-to-use-existing-tab-design-bd1c44484edf4917ba227f7dc923c4f6)

## Implementation description
* Update tabs on teacher dashboard to use the normal design

## Steps to test
1. View tabs

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by jfdoming** [comment]
image

File: frontend/src/components/teacher/dashboard/AssessmentsSection.tsx [hunk 1]
```diff
@@ -57,23 +57,14 @@ const AssessmentsSection = () => {
       )}
       {!!data?.length && !error && !loading && (
         <Tabs onChange={(index) => setCurrentTab(TEST_SESSION_STATUSES[index])}>
-          <TabList border="none" gap={8}>
+          <TabList>
```
Comment (carissa-tang) [inline frontend/src/components/teacher/dashboard/AssessmentsSection.tsx]: this tab code is now almost identical to `DisplayAssessmentsPage`. could we refactor that into a reusable component?

File: frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts [hunk 2]
```diff
@@ -4,9 +4,29 @@ import { useQuery } from "@apollo/client";
 import { GET_TEST_SESSIONS_BY_TEACHER_ID } from "../../../APIClients/queries/TestSessionQueries";
 import type { TestSessionOverviewData } from "../../../APIClients/types/TestSessionClientTypes";
 import AuthContext from "../../../contexts/AuthContext";
+import type { TestSessionStatus } from "../../../types/TestSessionTypes";
 import { getSessionTargetDate } from "../../../utils/TestSessionUtils";
 
-const useAssessmentDataQuery = (limit?: number) => {
+export type FormattedAssessmentData = {
```
Comment (carissa-tang) [inline frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts:10]: could we combine this type with [this one](https://github.com/uwblueprint/jump-math/blob/ee6fc5d82cc085d3515069ccdd2141c4a720bf27/frontend/src/components/teacher/view-sessions/TestSessionListItem.tsx#L16)? so we don't have duplicate

File: frontend/src/components/teacher/view-sessions/TestSessionListItemPopover.tsx [hunk 3]
```diff
@@ -29,19 +30,19 @@ const TestSessionListItemPopover = ({
...
 
   const [deleteTestSession] = useMutation(DELETE_TEST_SESSION, {
-    variables: { id: testSessionEditingData.testSessionId },
+    variables: { id: session.testSessionId },
     refetchQueries: [getQueryName(GET_TEST_SESSIONS_BY_TEACHER_ID)],
   });
 
   const onEditTestSession = () => {
-    history.push(Routes.DISTRIBUTE_ASSESSMENT_PAGE, testSessionEditingData);
+    history.push(Routes.DISTRIBUTE_ASSESSMENT_PAGE, session);
```
Comment (carissa-tang) [inline frontend/src/components/teacher/view-sessions/TestSessionListItemPopover.tsx:38]: if we're sending the whole session object, can we update this [line](https://github.com/uwblueprint/jump-math/blob/a874401de960ad06629997b6677bc12ea2bdfd73/frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx#L25)? also in that case we can delete the `TestSessionEditingData` type.

File: frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts [hunk 4]
```diff
@@ -4,9 +4,31 @@ import { useQuery } from "@apollo/client";
...
+  testName: string;
+  classroomId: string;
+  classroomName: string;
+  startDate: Date;
+  endDate: Date;
+  // Target date is the start date of the session UNLESS the session is
+  // active, in which case it is the end date
+  targetDate: Date;
+  status: TestSessionStatus;
+  accessCode: string;
```
Comment (carissa-tang) [inline frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts:22]: is this missing `notes` field?

File: frontend/src/components/teacher/view-sessions/TestSessionListItemPopover.tsx (hunk 3 above)
Comment (jfdoming) [reply frontend/src/components/teacher/view-sessions/TestSessionListItemPopover.tsx:38]: sure

File: frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts (hunk 4 above)
Comment (jfdoming) [reply frontend/src/components/teacher/view-sessions/useAssessmentDataQuery.ts:22]: yeah, nice catch

**Review by carissa-tang — APPROVED** [review]
nice!

**Pull request merged**